lib/index.js

```diff
@@ -592,7 +596,7 @@ class CouchbaseService {
 						},delay);
 					});
 				} else if (method === 'query'){
-					return this.cluster.query(...args, (error, result, meta) => {
+					return this.cluster.query(...args, {}, (error, result, meta) => {
```

Comment:
Wow... I assume this is the options parameter?

Reply:
That is correct